.travis.yml

```diff
@@ -1,30 +1,28 @@
 sudo: required
+services:
+  - docker
 
 language: node_js
+node_js:
+  - '6'
 
-services:
-  - docker
+cache:
+  directories:
+  - node_modules
+  - "$HOME/.cache/bower"
+  - "$HOME/.npm"
 
 before_install:
-  - npm install -g grunt-cli
+  - npm install -g grunt-cli bower
 
-before_script:
-  - if [ "$TRAVIS_BRANCH" == "master" ]; then
-    docker build --build-arg OPENSENSEMAP_API_URL=$OPENSENSEMAP_API_URL --build-arg OPENSENSEMAP_MAPTILES_URL=$OPENSENSEMAP_MAPTILES_URL -t senseboxdevs/osem .;
-    elif [ "$TRAVIS_BRANCH" == "development" ]; then
-    docker build --build-arg OPENSENSEMAP_API_URL=$OPENSENSEMAP_API_URL_DEV --build-arg OPENSENSEMAP_MAPTILES_URL=$OPENSENSEMAP_MAPTILES_URL_DEV -t senseboxdevs/osem:development .;
+script:
+  - if [[ "$TRAVIS_BRANCH" == "master" ]]; then
```

Comment:
Let`s use here: 
```bash
if ! [[ "$TRAVIS_BRANCH" == "master"]]
``` 
to check for all branches not equal to `master` and export the already existing `_DEV` travis environmental vairables
`OPENSENSEMAP_API_URL_DEV` and `OPENSENSEMAP_MAPTILES_URL_DEV`